fe/fe-core/src/main/java/com/starrocks/server/NodeMgr.java

```diff
@@ -920,6 +920,18 @@ public void replayDropFrontend(Frontend frontend) {
         }
     }
 
+    public boolean checkFeExistByRPCPort(String host, int rpcPort) {
+        try {
+            tryLock(true);
+            return frontends
+                    .values()
+                    .stream()
+                    .anyMatch(fe -> fe.getHost().equals(host) && fe.getRpcPort() == rpcPort);
```

Comment:
for fqdn situation, does this logic work?

Reply:
If the cluster is using FQDN, but the host specified in the external OLAP table is IP, this check will not work. But I think it's OK to keep this simple logic, because creating OLAP External table mapping to self is a rare case, and using FQDN but specified IP is rare too.